.github/workflows/upload-index.yml

```
@@ -27,5 +27,16 @@ jobs:
           ALGOLIA_APP_ID: ZPJB5I1QN7
           ALGOLIA_INDEX: prod_QUARTO
           QUARTO_INDEX_PATH: search.json
+          QUARTO_INDEX_URL: https://prerelease.quarto.org/search.json
```

This should be the released site, right?
```suggestion
          QUARTO_INDEX_URL: https://quarto.org/search.json
```

---

😓 yes ! 

Thanks for the review !